src/docs/guides/local-assets.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Local Assets
+description: Using Local Assets in your Components
+contributors:
+  - splitinfinities
+---
+
+# Local Assets
+
+Referencing Assets in Stencil can be tricky, but if you follow this guide you'll have no problem. 
+
+### Using Local Assets
+
+Lets dive in and look at an example of how to use local assets:
+
+Below is an example of your Component's folder structure containing the component, the stylesheet, and an assets directory. 
+
+```bash
+stencil-asset/
+  assets/
+    sunset.jpg
+    beach.jpg
+  stencil-asset.css
+  stencil-asset.tsx
+```
+
+Below is the stencil-asset.tsx file which will correctly load the assets based on a property called `src`. 
+
+```tsx
+import { Component, Prop, getAssetPath, h } from '@stencil/core';
+
+@Component({
+  tag: 'stencil-asset',
+  styleUrl: 'stencil-asset.css',
+  assetsDirs: ['assets']
+})
+export class StencilAsset {
+
+  @Prop() src: "sunset.jpg"|"beach.jpg" = "sunset.jpg";
+
+  render() {
+    <img src={`${getAssetPath('.')/assets/${this.asset}`} />
```

Comment:
```suggestion
    <img src={`${getAssetPath('.')assets/${this.asset}`} />
```

I think `getAssetPath(.)` returns a string ending with `/`?

Reply:
That's great to know!